packages/story-editor/src/components/form/color/color.js

```diff
@@ -141,7 +141,7 @@ const Color = forwardRef(function Color(
     <Container
       aria-label={containerLabel}
       isInDesignMenu={isInDesignMenu}
-      width={!ignoreSetWidth && width}
+      width={!ignoreSetWidth && width ? width : ''}
```

Comment:
With `width: PropTypes.number,` this is no longer needed.

Reply:
Or just return null instead of an empty string?

Reply:
```
  ["Warning: Received `%s` for a non-boolean attribute `%s`.·
    If you want to write it to the DOM, pass a string instead: %s=\"%s\" or %s={value.toString()}.·
    If you used to conditionally omit it with %s={condition && value}, pass %s={condition ? value : undefined} instead.%s", "false", "width", "width", "false", "width", "width", "width", "
```

Reply:
Is the width only needed for CSS or also in the DOM?
If the former then we might want to use transient props.